Resources/frontend/js/jquery.adyen-payment-selection.js

```diff
@@ -415,6 +413,22 @@
                     enableStoreDetails: this.__enableStoreDetails(paymentMethod)
                 }
             });
+        },
+        /**
+         * Create a minimal state when payment is handled by callback (e.g. PayPal payment)
+         * Use only when web components does NOT handle the payment
+         * @param payment
+         * @return {{data: {paymentMethod: {type}}}}
+         * @private
+         */
+        __buildMinimalState: function(payment) {
```

Comment:
Payment methods with redirect (like PayPal) do not get handled by web components, 
this method is merely to keep a single flow for setting the sessionStorage.